libcontainer/configs/validate/validator.go

```diff
@@ -80,6 +81,10 @@ func (v *ConfigValidator) security(config *configs.Config) error {
 		!config.Namespaces.Contains(configs.NEWNS) {
 		return fmt.Errorf("unable to restrict sys entries without a private MNT namespace")
 	}
+	if config.ProcessLabel != "" && !selinux.SelinuxEnabled() {
+		return fmt.Errorf("selinux label is specified in config, but selinux is disabled in kernel")
```

Comment:
This message could be confusing, because selinux is not enabled doesn't mean it's disabled in kernel, could be selinuxfs is not mounted, or it's disabled by semanage. Change it to
`selinux label is specified in config, but selinux is disabled or not supported` could be sufficient.